react/tic-tac-toe/src/app/screens/Game/index.js

```diff
@@ -1,20 +1,99 @@
 import React from 'react';
-
-import styles from './styles.module.scss';
 import Board from './components/Board';
 
-function Game () {
-  return (
-    <div className={styles.game}>
-      <div className={styles.gameBoard}>
-        <Board />
-      </div>
-      <div className={styles.gameInfo}>
-        <div>{/* status */}</div>
-        <ol>{/* TODO */}</ol>
+class Game extends React.Component {
+  constructor(props) {
```

Comment:
We aren't using constructor to initialize the state. You can just write 
state = { miState: 'something' } inside class definition